ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -24,6 +24,31 @@ scp "${SSHOPTS[@]}" /usr/bin/openshift-tests /usr/bin/kubectl "root@${IP}:/usr/l
 # Tests execution
 set +e
 
+# Mirroring test images is supported only for versions greater than or equal to 4.7
+# In such case the dev-scripts private reigstry is reused for mirroring the images.
+# Current openshift version is detected through dev-scripts
+# shellcheck disable=SC2046
+read -d '' OPENSHIFT_VERSION DEVSCRIPTS_REGISTRY DEVSCRIPTS_WORKING_DIR <<<$(ssh "${SSHOPTS[@]}" "root@${IP}" "source /root/dev-scripts/common.sh; source /root/dev-scripts/ocp_install_env.sh; cd /root/dev-scripts; echo \$(openshift_version); echo \$LOCAL_REGISTRY_DNS_NAME:\$LOCAL_REGISTRY_PORT; echo \$WORKING_DIR")
+
+TEST_ARGS=""
+if printf '%s\n%s' "4.7" "${OPENSHIFT_VERSION}" | sort -C -V; then
+  echo "### Mirroring test images"
+
+  DEVSCRIPTS_TEST_IMAGE_REPO=${DEVSCRIPTS_REGISTRY}/localimages/local-test-image  
+  # shellcheck disable=SC2087
+  ssh "${SSHOPTS[@]}" "root@${IP}" bash - << EOF
+set -xeuo pipefail
+
+source /home/afasano/dev-scripts/common.sh
```

Comment:
This path needs to be changed to $HOME?

Reply:
It's root right, maybe it will work better :) Thanks for spotting!